signingscript/script.py

```diff
@@ -136,15 +138,11 @@ def main(config_path=None):
         sslcontext = ssl.create_default_context(cafile=context.config['ssl_cert'])
         kwargs['ssl_context'] = sslcontext
     conn = aiohttp.TCPConnector(**kwargs)
-    with aiohttp.ClientSession(connector=conn) as session:
-        context.session = session
-        try:
-            loop.run_until_complete(async_main(context))
-        except ScriptWorkerTaskException as exc:
-            traceback.print_exc()
-            sys.exit(exc.exit_code)
-        finally:
-            loop.close()
```

Comment:
This was breaking all the pytest-asyncio tests, and my workaround of having a new `event_loop` fixture that opened a brand new event loop per function wasn't working.

We may figure out a different way to get these tests to work with new event loops, or we may not hit unclosed loop errors anymore.  Hoping for the latter.